addons/binding/org.openhab.binding.dsmr/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,34 @@
+Automatic-Module-Name: org.openhab.binding.dsmr
+Manifest-Version: 1.0
+Bundle-ActivationPolicy: lazy
+Bundle-ClassPath: .
+Bundle-ManifestVersion: 2
+Bundle-Name: DSMR Binding
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Bundle-SymbolicName: org.openhab.binding.dsmr;singleton:=true
+Bundle-Vendor: openHAB
+Bundle-Version: 2.4.0.qualifier
+Export-Package: org.openhab.binding.dsmr,
+ org.openhab.binding.dsmr.handler
+Import-Package: 
+ gnu.io,
```

Comment:
Would be great if you could refactor the implementation to use the [new ESH Serial API](https://github.com/eclipse/smarthome/tree/master/bundles/io/org.eclipse.smarthome.io.transport.serial/src/main/java/org/eclipse/smarthome/io/transport/serial) (which is identical to gnu.io, just uses its own namespace) as this will also directly enable the serial-over-tcp support.